Mage.Sets/src/mage/cards/a/ApproachOfTheSecondSun.java

```diff
@@ -64,28 +65,28 @@ public ApproachOfTheSecondSunEffect copy() {
     @Override
     public boolean apply(Game game, Ability source) {
         Player controller = game.getPlayer(source.getControllerId());
-        Spell spell = game.getStack().getSpell(source.getSourceId(), false);
-        if (controller != null && spell != null) {
-            ApproachOfTheSecondSunWatcher watcher
-                    = game.getState().getWatcher(ApproachOfTheSecondSunWatcher.class);
-            if (watcher != null
-                    && watcher.getApproachesCast(controller.getId()) > 1
-                    && spell.getFromZone() == Zone.HAND) {
-                // Win the game
-                controller.won(game);
-            } else {
-                // Gain 7 life and put this back into library.
-                controller.gainLife(7, game, source);
-
-                // Put this into the library as the 7th from the top
-                Card spellCard = game.getStack().getSpell(source.getSourceId(), false).getCard();
-                if (spellCard != null) {
+        if (controller == null)
+            return false;
```

Comment:
Please add braces for the if statement.

Reply:
Even around single line?

Reply:
That's the style I've seen everywhere else throughout the project.